### ISSUE TITLE
Deeplinks in a linux AppImage

### ISSUE THREAD

**Issue by liquidiert** [llfbandit/app_links#97]
**Describe the bug**

Hey, I tried to bundle my flutter application into an AppImage. The bundle works and also the custom scheme handler gets registered in the .desktop file. When I try to open the AppImage via my link it does not register that it got called.
App flow is the following:  
1.  Click on button that redirects to authentication page in browser
2. Browser calls custom scheme and opens app
3. Here it should register the call but it does not :(

I use [appimagetool](https://github.com/AppImage/AppImageKit/releases) to bundle the AppImage and [appimagelauncher](https://github.com/TheAssassin/AppImageLauncher) to include it into my system.

**Does it related to**

[ ] App Links (Android)  
[ ] Deep Links (Android)  
[ ] Universal Links (iOS)  
[ ] or Custom URL schemes? (iOS)  
[x] Custom URL schemes (Linux)

**Does the example project work?**

[ ] Yes  
[ ] No  
[x] Irrelevant here  

**Did you fully read the instructions for the targeted platform before submitting this issue?**

Uploaded your files to webserver, HTTPS, direct connection, scheme pattern setup, ...

[x] Yes  
[ ] No  
[ ] Irrelevant here

**Comment by llfbandit** [comment]
I'm not familiar with Linux but Linux code was refactored in version 6.0. It may worth a try.

**Comment by liquidiert** [comment]
@llfbandit I get an error using v6:
```text
══╡ EXCEPTION CAUGHT BY SERVICES LIBRARY ╞══════════════════════════════════════════════════════════
The following MissingPluginException was thrown while activating platform stream on channel
com.llfbandit.app_links/events:
MissingPluginException(No implementation found for method listen on channel
com.llfbandit.app_links/events)

When the exception was thrown, this was the stack:
#0      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:332:7)
<asynchronous suspension>
#1      EventChannel.receiveBroadcastStream.<anonymous closure> (package:flutter/src/services/platform_channel.dart:676:9)
<asynchronous suspension>
════════════════════════════════════════════════════════════════════════════════════════════════════
```
Should I open a new issue?

**Comment by llfbandit** [comment]
Sorry about that. 6.0.1 has been released to (hopefully) fix v6.0.0 init.

**Comment by liquidiert** [comment]
Hey @llfbandit 6.0.1 works again! I will try with appimages asap :)
